### PR TITLE
Add ephemeral Jenkins section

### PR DESCRIPTION
**Pull request by ramcguire**

### CODE REVIEW

File: docs/5/5.2-jenkins.md [hunk 1]
```diff
@@ -12,24 +12,23 @@ You should already be familiar with using the Jenkins web GUI interface however
...
-7. Update Jenkins plugins using the Jenkins CLI. Note: You may need to get creative here :)
-8. Clone and fork the spring-pet-clinic [project](https://github.com/liatrio/spring-petclinic).
-9. Download [petclinic-simple-config.xml](files/petclinic-simple-config.xml).
-10. Update the url field in the file to point to your Spring Pet Clinic fork.
-11. Create the Jenkins job and verify it was created.
+1. Setup a fresh Jenkins server with the recommended plugins.
+  * ensure you have jdk and Maven installed in your container
+ 
+ ?> Using the [official Jenkins image](https://hub.docker.com/r/jenkins/jenkins) will allow you to install plugins as part of the Docker build process.
+2. Create a `.yaml` file that configures a basic Jenkins instance and the recommended plugins installed in the previous step.
```
Comment (ChrisSchreiber) [inline docs/5/5.2-jenkins.md:19]: I think this should mention that the yaml configuration is specifically working with the Jenkins CasC plugin

File: docs/5/5.2-jenkins.md [hunk 2]
```diff
@@ -12,24 +12,23 @@ You should already be familiar with using the Jenkins web GUI interface however
...
+  * create a user with admin permissions
+  * make sure your instance has at least the git plugin, job-dsl plugin, and pipeline plugins installed
+  * check any warnings about permissions
+3. Using the [job-dsl plugin](https://jenkinsci.github.io/job-dsl-plugin/), add a Pipeline job to build spring-petclinic. You don't need to deploy this artifact.
```
Comment (ChrisSchreiber) [inline docs/5/5.2-jenkins.md:23]: Also I think worth calling out here that the job-dsl configuration is also part of the CasC yaml config

File: docs/5/5.2-jenkins.md [hunk 3]
```diff
@@ -12,24 +12,23 @@ You should already be familiar with using the Jenkins web GUI interface however
...
-
-1. Refer to the [Cli documentation](https://jenkins.io/doc/book/managing/cli/) and run the Jenkins CLI commands using password authentication and executing the command via SSH instead of the Java client.
-2. Experiment with other Jenkins commands.
 
 # Deliverable
 
-Discuss the importance of setting up Jenkins' jobs via a terminal with your group.
-Discuss the implications using SSH vs password authentication with the CLI.
+* Compare and contrast ephemeral Jenkins and non-ephemeral Jenkins.
+* Discuss the average windspeed velocity of a swallow.
```
Comment (ChrisSchreiber) [inline docs/5/5.2-jenkins.md:32]: 🤣

**Review by ChrisSchreiber — CHANGES_REQUESTED** [review]

File: docs/5/5.2-jenkins.md (hunk 1 above)
Comment (ramcguire) [reply docs/5/5.2-jenkins.md]: Specified that the .yaml file is for the Jenkins CasC plugin and linked to documentation

**Review by ChrisSchreiber — APPROVED** [review]

**Pull request merged**